jaksim/src/main/java/org/sopt/jaksim/mset/service/TaskMsetService.java

```diff
@@ -0,0 +1,21 @@
+package org.sopt.jaksim.mset.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.sopt.jaksim.mset.domain.CategoryMset;
+import org.sopt.jaksim.mset.domain.TaskMset;
+import org.sopt.jaksim.mset.repository.CategoryMsetRepository;
+import org.sopt.jaksim.mset.repository.TaskMsetRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class TaskMsetService {
+    private final TaskMsetRepository taskMsetRepository;
+    public List<TaskMset> getByTaskId(Long taskId) {
+        return taskMsetRepository.findByTaskId(taskId);
```

Comment:
taskId가 존재하지 않는 경우를 핸들링해주는 코드가 필요할 것 같아요 !
```suggestion
taskMsetRepository.findByTaskId(taskId).orElseThrow(
   () -> /* Error Handling */
)
```

Reply:
추가적으로, `orElseThrow()`를 쓰려면 repository의 코드를 수정해야할텐데요 ! 같이 고민해보면 좋을 것 같네요 :)

Reply:
말씀해주신대로 

```java
taskMsetRepository.findByTaskId(taskId).orElseThrow(
   () -> /* Error Handling */
)
```

이렇게 바꾸고 

repository는 

Optional을 추가해줘야할 것 같군요!